openfisca_web_api/model.py

```diff
@@ -38,7 +38,7 @@ def get_cached_or_new_decomposition_json(tax_benefit_system, xml_file_name = Non
         xml_file_name = tax_benefit_system.DEFAULT_DECOMP_FILE
     global decomposition_json_by_file_path_cache
     decomposition_json = decomposition_json_by_file_path_cache.get(xml_file_name)
-    if decomposition_json is None:
+    if decomposition_json is None and hasattr(tax_benefit_system, 'DECOMP_DIR'):
```

Comment:
Why would be the `get_cached_or_new_decomposition_json` function called if there is no `DECOMP_DIR` defined in the `tax_benefit_system`?

[environment.py](https://github.com/openfisca/openfisca-web-api/blob/7ae69fec1f9e8c76d5c41068878f402d22a0955c/openfisca_web_api/environment.py#L147-L148) calls this function checking for `DEFAULT_DECOMP_FILE`.

Where was the error triggered?